Gopkg.lock

```diff
@@ -70,7 +70,7 @@
   revision = "e89373fe6b4a7413d7acd6da1725b83ef713e6e4"
 
 [[projects]]
-  digest = "1:2e3c336fc7fde5c984d2841455a658a6d626450b1754a854b3b32e7a8f49a07a"
+  digest = "1:d2754cafcab0d22c13541618a8029a70a8959eb3525ff201fe971637e2274cd0"
```

Comment:
this is my own ignorance about how `go dep` works, but why did this file change when we didn't change any dependencies?

Reply:
it could have picked a newer version

Reply:
I believe it is due to including `github.com/ghodss/yaml` as a direct dependency inside the code rather than as a transitive dependency from the kubernetes packages

Reply:
kk, makes sense! thanks